AdofaiTweaks/Tweaks/KeyViewer/KeyViewer.cs

```diff
@@ -349,7 +350,7 @@ internal class KeyViewer : MonoBehaviour
 
         public void ClearCounts()
         {
-            foreach (KeyCode key in keyCounts.Keys)
+            foreach (KeyCode key in keyCounts.Keys.ToList())
```

Comment:
ToList() shouldn't have any effect here. I think the exception is related to keyCountTexts not containing the text object for the given key since those are only generated for the visible keys and the key counts are stored for pretty much all keys. A ContainsKey() should suffice.

Reply:
Actually the exception is related to the fact that I'm modifying keyCounts while enumerating through its keys, so it throws an exception.
Should I keep the ToList() Linq method or try to "cache" the keys as an array before the loop?

Reply:
Ah right in that case using ToList() works! I think the issue I mentioned can still occur though.